docs/bucket/replication/README.md

```diff
@@ -87,6 +87,7 @@ The replication configuration can now be added to the source bucket by applying
       "Status": "Enabled",
       "Priority": 1,
       "DeleteMarkerReplication": { "Status": "Disabled" },
+      "DeleteReplication": { "Status": "Disabled" },
```

Comment:
Can we mention this as a separate entity? i.e as part of 

### MinIO Extensions 

topic?